e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/ConstraintsTests.java

```diff
@@ -137,7 +137,7 @@ void constraintsSucceedNoViolations() throws IOException {
     hasura.deleteActivity(planId, activityId);
     hasura.awaitSimulation(planId);
     final var constraintsResults = hasura.checkConstraints(planId);
-    assertEquals(0, constraintsResults.size());
+    assertEquals(1, constraintsResults.size());
```

Comment:
What is this new result? Can you add asserts to describe it's properties?

Reply:
Done, Updated the test